frontend/utils/utils.js

```diff
@@ -200,6 +200,14 @@ var Utils = {
         return screen.width <= 960;
     },
 
+    /**
+     * This function indicate if the screen are in phone device size.
+     * @returns {boolean} True if the screen is smaller or equal to 450 pixels and false in otherwise.
+     */
+    isPhoneScreen: function isPhoneScreen() {
```

Comment:
your branch came from dev or responsive? If it came from responsive merge with it and then use isMobileScreen function, there this function receive the screenSize as parameter. If it came from dev, it's ok for me.